.github/workflows/publish-deb.yml

```diff
@@ -0,0 +1,89 @@
+name: Build and Publish Package
+
+on:
+  create
+
+jobs:
+  publish_apisix:
+    name: Build and Publish deb Package
+    runs-on: ubuntu-18.04
+    timeout-minutes: 60
+    env:
+      VAR_COS_BUCKET_CI: ${{ secrets.VAR_COS_BUCKET_CI }}
+      VAR_COS_BUCKET_REPO: ${{ secrets.VAR_COS_BUCKET_REPO }}
+      VAR_COS_ENDPOINT: ${{ secrets.VAR_COS_ENDPOINT }}
+      VAR_DEB_WORKBENCH_DIR: /tmp/output
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2.3.5
+        with:
+          submodules: recursive
+
+      - name: Init basic publish env
+        run: |
+          mkdir -p "${VAR_DEB_WORKBENCH_DIR}"
+          # init env var
+          TAG_DATE=$(date +%Y%m%d)
+          echo "TAG_DATE=${TAG_DATE}" >> "$GITHUB_ENV"
+
+      - name: Extract Tags name
+        if: ${{ startsWith(github.ref, 'refs/tags/') }}
+        id: tag_env
+        shell: bash
+        run: |
+          echo "##[set-output name=version;]$(echo ${GITHUB_REF##*/})"
+
+      - name: Extract Tags Type
+        if: ${{ startsWith(github.ref, 'refs/tags/') }}
+        id: tag_type
+        shell: bash
+        run: |
+          echo "##[set-output name=version;]$(echo ${GITHUB_REF#refs/tags/})"
+
+      - name: Build apisix deb Package
+        if: ${{ startsWith(steps.tag_type.outputs.version, 'apisix/') }}
+        env:
+          APISIX_TAG_VERSION: ${{ steps.tag_env.outputs.version}}
+        run: |
+          # build apisix
+          make package type=deb app=apisix openresty=apisix-base checkout=${APISIX_TAG_VERSION} version=${APISIX_TAG_VERSION} image_base=debian image_tag=bullseye-slim
+          mv ./output/apisix_${APISIX_TAG_VERSION}-0~debianbullseye-slim_amd64.deb ${VAR_DEB_WORKBENCH_DIR}
```

Comment:
Could we move `debianbullseye-slim` into a variable?

Reply:
Just add two env vars:
`
env:
   PACKAGE_OS: debian
   PACKAGE_OS_RELEASE: bullseye-slim
`
what do you think?

Reply:
@spacewander The definition of the package name is scattered, there is currently no unified function to define it, maybe we need another PR to solve it.

Reply:
OK